medium-tests/src/test/java/mediumtest/fixtures/ServerFixture.java

```diff
@@ -158,6 +158,11 @@ public ServerBuilder withPlugin(String pluginKey, UnaryOperator<ServerPluginBuil
       return this;
     }
 
+    public ServerBuilder withVersion(String version) {
+      this.version = version;
+      return this;
+    }
+
```

Comment:
Normally this information is needed only in the SQ context, that's why it's only part of the `newSonarQubeServer` method and not a `with` as usual

Reply:
It's not the case with the current flow. We read the version from the storage during sync. See other reply.